lib/commentOnClosedIssue.js

```diff
@@ -22,6 +23,9 @@ function commentOnClosedIssue(jsonResponse, headers) {
     Check.typeOf.object('jsonResponse', jsonResponse);
     Check.typeOf.object('headers', headers);
 
+    if (defined(jsonResponse.pull_request)) {
+        return commentOnClosedIssue._implementation(jsonResponse.pull_request.url, jsonResponse.pull_request.comments_url, headers);
```

Comment:
Rather than duplicate the function call, pull the variables out into locals.  This will make it easier to maintain.